bot_globals.py

```diff
@@ -36,6 +36,26 @@
 
 RATINGS = {}
 
+MILESTONE_ROLES = {
+    1: ("Initiate (1 Point)", discord.Color.blue()),          # 1 Problem
+    10: ("Problem Solver (10 Points)", discord.Color.green()),  # 10 Problems
+    25: ("Algorithm Apprentice (25 Points)", discord.Color.gold()),  # 25 Problems
+    50: ("Logic Guru (50 Points)", discord.Color.purple()),     # 50 Problems
+    75: ("Code Crusader (75 Points)", discord.Color.orange()),  # 75 Problems
+    100: ("LeetCode Legend (100 Points)", discord.Color.red()),  # 100 Problems
+    150: ("Problem Connoisseur (150 Points)", discord.Color.teal()),  # 150 Problems
+    200: ("Mastermind (200 Points)", discord.Color.dark_blue())  # 200 Problems
+}
+
+STREAK_ROLES = {
+    3: ("Streak Starter (3 Days)", discord.Color.blue()),          # 3 Days
+    7: ("Streak Challenger (7 Days)", discord.Color.green()),  # 7 Days
+    14: ("Streak Conqueror (14 Days)", discord.Color.gold()),  # 14 Days
+    30: ("Streak Legend (30 Days)", discord.Color.purple()),     # 30 Days
+}
+
+VERIFIED_ROLE = "CodeGrind Verified"
+
```

Comment:
I'm not sure I agree with the decided milestone values and names. I think it should be something we discuss on discord

Reply:
Agreed, these are just placeholder ones